tests/ui/parser/recover/recover-field-semi.stderr

```diff
@@ -1,35 +1,27 @@
 error: struct fields are separated by `,`
   --> $DIR/recover-field-semi.rs:2:13
    |
-LL | struct Foo {
-   |        --- while parsing this struct
```

Comment:
You could recover this label by passing down the adt span and kind into this field parsing function. I don't think it's very hard to do.

Reply:
do you mind trying this?

Reply:
Done!